### PR TITLE
ci: fix tests can not update in pr

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

# Description

If we change "dapr.yml" to add/change tests, now it would not run new tests since skip.

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by beiwei30 — COMMENTED** [review]
This change looks good to me except there's no issue to track.

File: .github/workflows/dapr-e2e-kind.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
       id: skip_check
       uses: fkirc/skip-duplicate-actions@v3.4.0
       with:
-        paths_ignore: '["**.md", ".codecov.yaml"]'
+        paths_ignore: '["**.md", ".codecov.yaml", ".github/ISSUE_TEMPLATE/*"]'
```
Comment (berndverst) [inline .github/workflows/dapr-e2e-kind.yml:83]: Can you also add `.github/**/**.yml` ?

```suggestion
        paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml", ".github/ISSUE_TEMPLATE/*"]'
```

File: .github/workflows/dapr.yml [hunk 2]
```diff
@@ -27,7 +27,7 @@ jobs:
         uses: fkirc/skip-duplicate-actions@v3.4.0
         with:
           cancel_others: 'true'
-          paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml"]'
+          paths_ignore: '["**.md", ".codecov.yaml", ".github/ISSUE_TEMPLATE/*"]'
```
Comment (berndverst) [inline .github/workflows/dapr.yml:30]: Please add `.github/**/**.yml` again

```suggestion
          paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml", ".github/ISSUE_TEMPLATE/*"]'
```

File: .github/workflows/dapr.yml [hunk 3]
```diff
@@ -66,7 +66,7 @@ jobs:
...
```
Comment (berndverst) [inline .github/workflows/dapr.yml:69]: Please add `.github/**/**.yml` again

```suggestion
          paths_ignore: '["**.md", ".codecov.yaml", ".github/**/**.yml", ".github/ISSUE_TEMPLATE/*"]'
```

File: .github/workflows/dapr-e2e-kind.yml (hunk 1 above)
Comment (artursouza) [reply .github/workflows/dapr-e2e-kind.yml:83]: I agree that removing the `yml` path is not ideal.

File: .github/workflows/dapr-e2e-apps.yml [hunk 4]
```diff
@@ -12,6 +12,17 @@ on:
       - master
       - release-*
 jobs:
+  pre_job:
+    name: Skip Duplicate Actions
```
Comment (artursouza) [inline .github/workflows/dapr-e2e-apps.yml:16]: Please, add condition to only skip in pull requests. Same for the other files, please.

**Review by artursouza — CHANGES_REQUESTED** [review]

**Pull request closed without merging**